packages/react-scripts/scripts/start.js

```diff
@@ -46,7 +46,7 @@ const {
 const openBrowser = require('react-dev-utils/openBrowser');
 const paths = require('../config/paths');
 const configFactory = require('../config/webpack.config');
-const createDevServerConfig = require('../config/webpackDevServer.config');
+const createDevServerConfig = require('../config/WebpackDevServer.config');
```

Comment:
Be careful with find and replace. Please revert this change. We dont want to rename any file here

Reply:
Ahh right!